### PR TITLE
PLT-4554 [FE]Sell a subscription

### PR DESCRIPTION
**Pull request by amnambiar**
https://input-output.atlassian.net/browse/PLT-4554

### CODE REVIEW

**Comment by RSoulatIOHK** [comment]
I think it still needs to have an actual effect when clicking on the different "pay" buttons to close PLT-4554. But it looks good. You should wait for the more expert review from Bogdan, this was just my 2 cents.

File: react-web/src/pages/subscription/Subscription.tsx [hunk 1]
```diff
@@ -1,9 +1,88 @@
 import React from "react";
-import MaintenancePage from "pages/maintenance/Maintenance";
+import PricingCard from "./components/PricingCard/PricingCard";
+import "./Subscription.scss";
 
+const developerTiers = [
```
Comment (bogdan-manole) [inline react-web/src/pages/subscription/Subscription.tsx:5]: Shouldn't we get the descriptions from `GET /tiers`?

File: react-web/src/pages/subscription/components/PricingCard/PricingCard.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import "./PricingCard.scss";
+
+const PricingCard: React.FC<any> = ({ name, disabled, featureSet, description, price }) => {
```
Comment (bogdan-manole) [inline react-web/src/pages/subscription/components/PricingCard/PricingCard.tsx:4]: why `any` ?

File: react-web/src/pages/subscription/components/PricingCard/PricingCard.tsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+            </span>          
+          </div>
+        </> : null}
+        
+      </div>
+
+      <div className="card_feature_list">
+        <ul>
+          <li>{featureSet}</li>
+          {/* <li>
```
Comment (bogdan-manole) [inline react-web/src/pages/subscription/components/PricingCard/PricingCard.tsx:29]: do we need the comment?

**Review by bogdan-manole — CHANGES_REQUESTED** [review]

**Comment by amnambiar** [comment]
Closing this PR as most recent changes are included within https://github.com/input-output-hk/dapps-certification/pull/64

**Pull request closed without merging**